crates/fm/src/file_reader.rs

```diff
@@ -26,7 +26,8 @@ cfg_if::cfg_if! {
         pub(crate) fn read_file_to_string(path_to_file: &Path) -> Result<String, Error> {
             use std::io::ErrorKind;
 
-            match StdLibAssets::get(path_to_file.to_str().unwrap()) {
+            let path_str = path_to_file.to_str().unwrap();
+            match StdLibAssets::get(path_str) {
```

Comment:
Why was this change made? It's inconsistent with the non-wasm implementation.

Reply:
Would not compile otherwise. Out of previous bad merge?